### ISSUE TITLE
No tooltip display for ToolStripDropDownButton after using right/Left/Enter to collapse ToolStripDropDownButton

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#12917]
### .NET version

.NET SDK 10.0.100-preview2.25110.18

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Can reproduce on all previous, NET versions

### Issue description

No tooltip display for ToolStripDropDownButton after using right/Left/Enter to collapse ToolStripDropDownButton 
![Image](https://github.com/user-attachments/assets/27a4b04c-8c25-478f-bfde-304230cedf51)

### Steps to reproduce

1. Create Winforms .NET app with ToolStrip control 
2. Add ToolStripDropDownButton item for toolStip1, and add toolStripMenuItem for ToolStripDropDownButton  
3. Set the toolstrip1's TabStop property to True 
4. Build and run app 
5. Press down/Enter to expand ToolStripDropDownButton in toolStrip1 - **it will focus on toolStripMenuIteml** 
6. Press right/Left/Enter to collapse ToolStripDropDownButton in toolStrip1- **it will focus on ToolStripDropDownButton, but no tooltip display for ToolStripDropDownButton**  

**More info:**
The issue cannot reproduce when using Alt keyboard to collapse the ToolStripDropDownButton

**Comment by merriemcgaw** [comment]
Closing this as it's not a regression from Framework.
